### ISSUE TITLE
Correction to gan_mnist d_loss_fake labels

### ISSUE THREAD

**Issue by jg1141** [udacity/deep-learning#135]
In the [solution Jupyter notebook for gan_mist](https://github.com/udacity/deep-learning/blob/master/gan_mnist/Intro_to_GANs_Solution.ipynb), cell 14, line 7,
`d_loss_fake = tf.reduce_mean(
                  tf.nn.sigmoid_cross_entropy_with_logits(logits=d_logits_fake, 
                                                          labels=tf.zeros_like(d_logits_real)))`
should have  a zero labels tensor like the **fake** logits, not the **real** ones:
`d_loss_fake = tf.reduce_mean(
                  tf.nn.sigmoid_cross_entropy_with_logits(logits=d_logits_fake, 
                                                          labels=tf.zeros_like(d_logits_fake)))`

It was just luck that the fake and real logit tensors have the same length, so the code manages to work as is.

**Comment by huseinzol05** [comment]
of course it will be same.

**Comment by abhiojha8** [comment]
This has been resolved via PR #238
